### PR TITLE
macOS app naming improvements

### PR DESCRIPTION
**Pull request by i0ntempest**
Sets "SlimeVR" as app name in multiple locations. In the future this could be replaced by `displayName` if [tauri ever implement it](https://github.com/tauri-apps/tauri/issues/8109).

### CODE REVIEW

**Review by ImUrX — CHANGES_REQUESTED** [review]
The SlimeVR Server is not the GUI, and the app is specifically the GUI

Please change it to SlimeVR

**Comment by i0ntempest** [comment]
> The SlimeVR Server is not the GUI, and the app is specifically the GUI

Okay, but the windows installer installs a “SlimeVR Server” shortcut into start menu that launches the GUI, so I wonder why is it not appropriate here? It’s the name for the package that contains both GUI and server.

**Comment by ImUrX** [comment]
And we also have the linux desktop file which is just `SlimeVR`, we have the binary called `slimevr.exe` in Windows.

I don't really know the reason for why it's called `SlimeVR Server` in the start menu but I suspect it's been like that since the start and having to change it seems like too much problem.

**Comment by ImUrX** [comment]
why are you signing the app

**Comment by i0ntempest** [comment]
> why are you signing the app

Do you not want that? I was just about to say I added that because without a signature users are probably going to have problems launching the app because macOS Gatekeeper is only getting stricter.

**Comment by ImUrX** [comment]
I mean, we don't have any signature to sign it with XD. Or does every macOS come with a user signature?

**Comment by i0ntempest** [comment]
> I mean, we don't have any signature to sign it with XD. Or does every macOS come with a user signature?

I signed with `-` identity which is local adhoc signature. It's not a developer identity but should be better than flat out no signature.

Also idealy version of the app bundle should match github releases, right now I believe it's inherting the GUI version. Do you want that to be overwritten in Info.plist as well?

**Comment by ImUrX** [comment]
then its fine to sign it.

the version stuff, it would be nice to fix the actual issue, the `package.json` of `gui` is not the same as the git tag and i don't really know how to fix it

**Comment by i0ntempest** [comment]
> the version stuff, it would be nice to fix the actual issue, the `package.json` of `gui` is not the same as the git tag and i don't really know how to fix it

It's possible with `npm version`, but it's usually used to create/manage git tags, not the other way around (following existing git tags). I'm experimenting with it now, there should be a way around.

**Comment by i0ntempest** [comment]
I kinda figured it out. Workflows triggered by releases will have their version in `gui/packages.json` corrected but that will also make the build dirty. To truely get around this I guess you really need to use `npm version` the way it meant to be used.

File: .github/workflows/gradle.yaml [hunk 1]
```diff
@@ -256,23 +256,37 @@ jobs:
...
           rustup target add aarch64-apple-darwin
+          tag=${GITHUB_REF#refs/*/}
+          if [[ "$tag" == v* ]]; then
+            echo "Overwriting version: $tag"
+            cd gui
+            pnpm version "$tag" --force --allow-same-version --workspaces-update false
+            git add package.json
+            cd ..
+            pnpm version "$tag" --force --allow-same-version --workspaces-update false
+          fi
```
Comment (ImUrX) [inline .github/workflows/gradle.yaml:267]:
```suggestion
```
i have another way to fix this, but it's gonna be in another PR.
Mind removing this? :3

**Review by ImUrX — CHANGES_REQUESTED** [review]
let's leave this for another PR

**Review by ImUrX — APPROVED** [review]

**Pull request merged**